### ISSUE TITLE
Needs action for completed missions

### ISSUE THREAD

**Issue by Benjamin-Berger** [Detergent13/tft-bot#7]
There are many missions such as: Play 5 games. Theses will complete and the bot will have no way of moving past these screens. 

Its a very small issue and after enough missions are completed, they go away. Thanks for you work!

**Comment by Detergent13** [comment]
Actually already has a check on lines 135-136,

`
    while onscreen("./captures/missions ok.png"):
        click_to("./captures/missions ok.png")`

But let me know if that's not working correctly for you!

**Comment by Benjamin-Berger** [comment]
Sorry I just saw that in the code and was coming to edit my comment. Unfortunately it was not working. I don't have any more missions to test though at the moment.

**Comment by Detergent13** [comment]
> Sorry I just saw that in the code and was coming to edit my comment. Unfortunately it was not working. I don't have any more missions to test though at the moment.

Gotcha, not sure why that's not working, might be a resolution issue (client should be 1280x720) or just that you need to recapture the image/increase the tolerance? If you happen to run into it again, try grabbing a screenshot of the button and replacing it.

**Comment by Detergent13** [comment]
I've never had an issue with it that I can remember, but definitely possible that it's just finicky for some people.

**Comment by Benjamin-Berger** [comment]
tl;dr; indent the time.sleep(5) at line 137.

Tested on an alt account and it worked there. One possible answer for why it didn't work before is because of multiple missions finishing at the same time?

I'm not totally sure this is the case, but if finishing multiple missions happens like this:
1. mission shows up
2. ok button to confirm mission finish shows
3. click ok button
4. repeat from one until no more missions

Then it makes sense the code would break here. The while on screen at line 135 checks for the mission and immediately presses ok. If the second missions "ok" button doesn't show up immediately then the while loop will end, but there will be no way to progress. 

An easy fix is to indent the time.sleep(5) at line 137 and make it part of the above while loop.

But again that's only if multiple missions works like this. Otherwise i would just chalk it up to user error and close this bug.

**Comment by Detergent13** [comment]
Done- not entirely sure if multiple missions can cause issues, but no harm in moving the delay either way.

[b9ae3ed](https://github.com/Detergent13/tft-bot/commit/b9ae3edd4d6043daaf149e0ab1c83aa633766a65)
